src/components/DashBoard.vue

```diff
@@ -216,50 +252,48 @@ body.sb-sidenav-toggled #wrapper #sidebar-wrapper {
 }
 
 .dashboard {
-  color: #FA6731;
+  color: #fa6731;
   letter-spacing: 0.15px;
   line-height: 6px;
   margin-top: -1px;
   min-width: 42px;
   white-space: nowrap;
-  
 }
 
-.dashboard-menu{
-  border:none !important;
-  padding:15px !important;
-  text-decoration:none;
-  color:black;
+.dashboard-menu {
+  border: none !important;
+  padding: 15px !important;
+  text-decoration: none;
+  color: black;
 }
-.navbar-account-img{
-  height:35px;
-  width:35px;
-  border:1px solid grey;
+.navbar-account-img {
+  height: 35px;
+  width: 35px;
+  border: 1px solid grey;
   border-radius: 50%;
-
 }
-.nav-link{
-  position:relative !important;
+.nav-link {
+  position: relative !important;
```

Comment:
Why is it required to add "!important"? Can't we remove it?

Reply:
We're adding !important property to override the predefined bootstrap class properties